### DISCUSSION TITLE
Logged in github.com main page is confusing

### DISCUSSION THREAD

**Discussion by ralphtherapper** [community/community#13965] [Other Feature Feedback, Questions, & Ideas]
Hi,

I am writing this ticket on behalf of several people in my team. We do work on github repositories and we are part of organization hence all our repositories basically lie under organisation.

What happens:
When you type in github.com on your browser and land on your "dashboard". It lists repository list on left side that list random repositories. It states "recent" repositories even the listen repositories are something else.

On same list there is search field that does not find the repositories you are working on.

1) if you list "recent repositories list" make sure it list RECENT repositories and not random repositories.

2) If you offer search functionality on any page make sure it SEARCHES valid database.

**Comment by zjeffer** [comment]
The first one bothers me the most. In my case, the top repo is one of my archived repositories with last activity 4 months ago. 
Below that, a non-archived repo that I often visit (this is fine but maybe shouldn't be so high up)

The rest of the repositories that are visible without clicking "show more", are random repos that haven't had activity in weeks. 
If I click "show more", a huge list of repos appears, some of which I have created yesterday, some of which I haven't visited in months. The list seems to be in random order.

In the past 10 days, I created 4 new repositories (including 2 forks). None of them are visible without clicking "show more", and even then they are below old repositories that heven't been updated in months. How does it decide the order of the repositories?

**Comment by omer-os** [comment]
yup very true. i want to see most recent repos when i open the page first time. **github should fix this !**
